frontend/lib/shared/animations/invalid_number_pad_input_animation.dart

```diff
@@ -0,0 +1,76 @@
+import 'package:flutter/material.dart';
+
+class InvalidNumberPadInputAnimation extends StatefulWidget {
+  final String textValue;
+  final bool shouldAnimate;
+  final TextStyle? textStyle;
+
+  const InvalidNumberPadInputAnimation(
+      {required this.textValue,
+      required this.shouldAnimate,
+      this.textStyle,
+      super.key});
+
+  @override
+  State<InvalidNumberPadInputAnimation> createState() =>
+      _InvalidNumberPadInputAnimationState();
+}
+
+class _InvalidNumberPadInputAnimationState
+    extends State<InvalidNumberPadInputAnimation>
+    with SingleTickerProviderStateMixin {
+  late final AnimationController _controller = AnimationController(
+    duration: const Duration(milliseconds: 375),
```

Comment:
duration here feels a tad bit long maybe ~100 milliseconds instead?

Reply:
@kirahsapong yea that works! we can worry about tweaking stuff like animations later on